web/libs/editor/src/mixins/HighlightMixin.js

```diff
@@ -1,21 +1,19 @@
 import { types } from "mobx-state-tree";
 
 import Utils from "../utils";
-import { guidGenerator } from "../utils/unique";
-import Constants, { defaultStyle } from "../core/Constants";
+import { defaultStyle } from "../core/Constants";
 import { isDefined } from "../utils/utilities";
-import { FF_LSDV_4620_3, isFF } from "../utils/feature-flags";
 
 const HIGHLIGHT_CN = "htx-highlight";
 const HIGHLIGHT_NO_LABEL_CN = "htx-no-label";
-const IDENTIFIER_LENGTH = 5;
 const LABEL_COLOR_ALPHA = 0.3;
 
 export const HighlightMixin = types
   .model()
   .views((self) => ({
     get _hasSpans() {
       // @todo is it possible that only some spans are connected?
+      // @TODO: Is it a legacy code we no need anymore?
```

Comment:
```suggestion
      // @TODO: Need to check if it is still necessary condition. The way of working with spans was changed and it could affect this part. The main question, is there still a way to get `isConnected === false`
```